### PR TITLE
change ui forms

### LINKED ISSUE #60: change forms ui
Reduce the size of the forms and improve the design.

### PR DESCRIPTION
**Pull request by Vova-Romaniuk**

### CODE REVIEW

File: StudCity.Frontend/src/App.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ const router = createBrowserRouter([
 		element: <Registration />
 	},
 	{
-		path: '/not-found',
+		path: "*",
 		element: <NotFound/>
```
Comment (maryanchuk04) [inline StudCity.Frontend/src/App.js]: Add space inside call component

File: StudCity.Frontend/src/pages/Authenticate/index.jsx [hunk 2]
```diff
@@ -29,8 +29,8 @@ function Authenticate() {
 	return (
 		<div className="w-1/1 h-screen flex">
 			<div className="w-1/2 h-screen flex bg-primatyWhite">
-				<div className="container flex flex-col w-4/5 m-auto h-[85%] bg-primatyWhite rounded-3xl p-11 shadow-md ">
-					<h1 className="w-full text-center font-bold text-4xl my-3 text-primaryAuthentication">Welcome back</h1>
+				<div className="container flex flex-col w-2/3 m-auto h-[80%] justify-between bg-primatyWhite rounded-3xl p-11 shadow-md ">
```
Comment (maryanchuk04) [inline StudCity.Frontend/src/pages/Authenticate/index.jsx]: Maybe we can use h-4/5 ? @Serhiy-dotcom @Vova-Romaniuk

Comment (Serhiy-dotcom) [reply StudCity.Frontend/src/pages/Authenticate/index.jsx]: Yes great suggestion, docs says we definitely can

File: StudCity.Frontend/src/UI/GoogleButton.jsx [hunk 3]
```diff
@@ -2,7 +2,7 @@ import React from "react";
 
 function GoogleButton({ className = "" }) {
     return (
-		<button type='button' className={`flex break-inside bg-white text-primaryAuthentication border-2 border-primaryAuthentication rounded-3xl px-6 py-3 mb-4 w-full h-16 ${className}`}>
+		<button type='button' className={`flex break-inside bg-white text-primaryAuthentication border-2 border-primaryAuthentication rounded-3xl w-full mx-auto mb-4 w-full h-12 ${className}`}>
 			<div className='m-auto'>
 				<div className='flex items-center justify-start flex-1 space-x-4 text-xl'>
 					<svg width='25' height='25' viewBox='0 0 24 24'>
```
Comment (Serhiy-dotcom) [inline StudCity.Frontend/src/UI/GoogleButton.jsx]: Please use Svg component

**Review by Serhiy-dotcom — CHANGES_REQUESTED** [review]

**Review by Serhiy-dotcom — APPROVED** [review]

**Review by maryanchuk04 — APPROVED** [review]
LGTM

**Pull request merged**